uwblueprint/sistering
# Remove signup resolvers

## Ticket link

Closes #275



## Implementation description
* Removes all implementations of createShiftSignup and updateShiftSignup in codebase 
* replaces use of updateshiftsignup with upsertDeleteShiftSignups in RemoveVolunteerModal.js


## Steps to test
1. Navigate to http://localhost:3000/admin/schedule/posting/1
2. Click on the shift from 1:30-2:30pm on July 27th
3. Beside available volunteers in the side panel, click "edit", select valorie volunteer and click save
4. Click review, and remove Valerie volunteer from Wednesday, July 27th 
5. Click back to editing, and verify Valerie volunteer is properly removed
![Screen Shot 2022-07-02 at 6 51 18 PM](https://user-images.githubusercontent.com/82241706/177018172-be9df13c-5373-4fdf-a470-0528479a60b3.png)


## What should reviewers focus on?
* all implementations of createShiftSignup and updateShiftSignup are removed without creating new bugs

## Checklist
